### DISCUSSION TITLE
[rollup-plugin-windicss] can not configure windicss with rollup. please help.

### DISCUSSION THREAD

**Discussion by MadadiSaeed** [windicss/vite-plugin-windicss#152] [Q&A]
I am trying to create rollup-solid-windicss starter project with guide you put in this github page. But I can't make it work. please healp me. Thanks.
this is my plugins list in `rollup.config.js` file and I am adding `import 'virtual:windi.css'` for my `index.js` entry file, but I am getting an error.

    [
	    windicss(),
	    postcss(postcssOpts),
	    replace(replaceOpts),
	    resolve(resolveOpts), // node-resolve
	    babel(babelOpts),
	    commonjs(commonjsOpts)
    ]

error: 

    (!) Unresolved dependencies
    https://rollupjs.org/guide/en/#warning-treating-module-as-external-dependency
    virtual:windi.css (imported by src/index.js)

**Comment by hannoeru** [comment]
Can you provide more context? or try `windicss.default()`

**Reply by MadadiSaeed** [reply]
here is my simple project. i tried to add windicss plugin but i was not successful. [https://gitlab.com/samadadi/solid-ssr-stream-exmaple](https://gitlab.com/samadadi/solid-ssr-stream-exmaple)
